### DISCUSSION TITLE
Can I push to remote with different path?

### DISCUSSION THREAD

**Discussion by BigFaceBoy** [treeverse/dvc#10393] [General]
I want to track my different platform file in my repo, and I don`t want to add remote with every platform.Is there any way that i can push files to different path, and I just add remote with  bucket root path?

**Comment by dberenbaum** [comment]
DVC usually tracks the file inside your repo and relative to the repo directory, so it should not be platform-specific. Then you configure one or multiple remote storage locations where you push backups of those files.
